### PR TITLE
pages.*: sync page titles

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
This PR is the result of the newly introduced Python-script in #12732 

Edit: 
As long as #12732 is not merged, this is just for showcasing the output.

### CODE REVIEW

**Review by Magrid0 — APPROVED** [review]

**Review by vitorhcl — APPROVED** [review]

File: pages.es/common/fossil-delete.md
Comment (kant) [inline pages.es/common/fossil-delete.md:1]: LGTM

File: pages.es/common/fossil-forget.md
Comment (kant) [inline pages.es/common/fossil-forget.md:1]: LGTM

File: pages.es/common/fossil-new.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# fossil-new
+# fossil new
 
 > Este comando es un alias de `fossil-init`.
```
Comment (kant) [inline pages.es/common/fossil-new.md:3]: > Este comando es un alias de `fossil init`.

File: pages.es/common/gh-cs.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# gh-cs
+# gh cs
 
 > Este comando es un alias de `gh-codespace`.
```
Comment (kant) [inline pages.es/common/gh-cs.md:3]: > Este comando es un alias de `gh codespace`.

File: pages.es/common/pio-init.md
Comment (kant) [inline pages.es/common/pio-init.md:1]: LGTM

File: pages.es/common/tlmgr-arch.md
Comment (kant) [inline pages.es/common/tlmgr-arch.md:1]: LGTM

File: pages.es/common/type.md
Comment (kant) [inline pages.es/common/type.md:1]: LGTM

File: pages.es/linux/ip-route-list.md
Comment (kant) [inline pages.es/linux/ip-route-list.md:1]: > Este comando es un alias de `ip route show`.

**Review by kant — CHANGES_REQUESTED** [review]
Some comments, some LGTM

**Review by noraj — APPROVED** [review]
LGTM for the titles of French pages

**Review by gutjuri — APPROVED** [review]

File: pages.pl/common/clamav.md
Comment (acuteenvy) [inline pages.pl/common/clamav.md:1]: It's not just the title, the entire page is misleading. This isn't even a command.

These pages with mismatched titles, it all looks like #11373 again. [It's an automatically generated page with `set-alias-page.py`](https://github.com/tldr-pages/tldr/commits/main/pages.pl/common/clamav.md), which as of now makes a whole lot of mistakes when translating pages.

**Review by acuteenvy — APPROVED** [review]

File: pages.pl/common/fossil-ci.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# fossil-ci
+# fossil ci
 
 > To polecenie jest aliasem `fossil-commit`.
```
Comment (acuteenvy) [inline pages.pl/common/fossil-ci.md:3]: Hyphens haven't been removed from descriptions. This is also the case with other languages.
```suggestion
> To polecenie jest aliasem `fossil commit`.
```

File: pages.pl/common/fossil-new.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# fossil-new
+# fossil new
 
 > To polecenie jest aliasem `fossil-init`.
```
Comment (acuteenvy) [inline pages.pl/common/fossil-new.md:3]:
```suggestion
> To polecenie jest aliasem `fossil init`.
```

File: pages.pl/common/gh-cs.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# gh-cs
+# gh cs
 
 > To polecenie jest aliasem `gh-codespace`.
```
Comment (acuteenvy) [inline pages.pl/common/gh-cs.md:3]:
```suggestion
> To polecenie jest aliasem `gh codespace`.
```

File: pages.pl/linux/ip-route-list.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# ip-route-list
+# ip route list
 
 > To polecenie jest aliasem `ip-route-show`.
```
Comment (acuteenvy) [inline pages.pl/linux/ip-route-list.md:3]:
```suggestion
> To polecenie jest aliasem `ip route show`.
```

**Comment by acuteenvy** [comment]
There are changes from #12732 here, could you please `git revert` that? (https://github.com/tldr-pages/tldr/pull/12734/commits/f7d3c18a7d38143f4424de51b093f5c81bf9228d)

**Pull request merged**